src/Bicep.LangServer.IntegrationTests/CodeActionTests.cs

```diff
@@ -95,48 +95,65 @@ public async Task RequestingCodeActionWithFixableDiagnosticsShouldProduceQuickFi
 
             // construct a parallel compilation
             var lineStarts = compilation.SourceFileGrouping.EntryPoint.LineStarts;
-            var fixables = compilation.GetEntrypointSemanticModel().GetAllDiagnostics().OfType<IFixable>();
+            var fixables = compilation.GetEntrypointSemanticModel().GetAllDiagnostics().OfType<IFixable>().ToList();
 
             foreach (var fixable in fixables)
             {
                 foreach (var span in GetOverlappingSpans(fixable.Span))
                 {
+                    var range = span.ToRange(lineStarts);
                     var quickFixes = await client.RequestCodeAction(new CodeActionParams
                     {
                         TextDocument = new TextDocumentIdentifier(uri),
-                        Range = span.ToRange(lineStarts),
+                        Range = range,
                     });
 
                     // Assert.
                     quickFixes.Should().NotBeNull();
 
+                    var spansOverlap = (IFixable f) =>
+                    {
+                        if (span.Position <= f.Span.Position)
```

Comment:
Can you use this method here?
https://github.com/Azure/bicep/blob/91f46be65b1e32cfb50782e205a20a10acf89481/src/Bicep.Core/Parsing/TextSpan.cs#L121

Reply:
I can't use that method alone, though I should probably rename the local method something like `spansOverlapOrAbut`. The equivalent using `TextSpan` methods would be `TextSpan.AreOverlapping(span, f.Span) || TextSpan.AreNeighbors(span, f.Span) || TextSpan.AreNeighbors(f.Span, span)`

Reply:
Makes sense!